homeassistant/components/blue_current/sensor.py

```diff
@@ -0,0 +1,342 @@
+"""Support for Blue Current sensors."""
+from __future__ import annotations
+
+from homeassistant.components.sensor import (
+    SensorDeviceClass,
+    SensorEntity,
+    SensorEntityDescription,
+    SensorStateClass,
+)
+from homeassistant.config_entries import ConfigEntry
+from homeassistant.const import (
+    UnitOfElectricCurrent,
+    UnitOfElectricPotential,
+    UnitOfEnergy,
+    UnitOfPower,
+)
+from homeassistant.core import HomeAssistant, callback
+from homeassistant.helpers.dispatcher import async_dispatcher_connect
+from homeassistant.helpers.entity_platform import AddEntitiesCallback
+
+from . import Connector
+from .const import DOMAIN
+from .entity import BlueCurrentEntity
+
+TIMESTAMP_KEYS = ("start_datetime", "stop_datetime", "offline_since")
+
+SENSORS = (
+    SensorEntityDescription(
+        key="actual_v1",
+        native_unit_of_measurement=UnitOfElectricPotential.VOLT,
+        device_class=SensorDeviceClass.VOLTAGE,
+        name="Voltage Phase 1",
```

Comment:
I would suggest using entity translations, for an example change of what should happen checkout https://github.com/home-assistant/core/pull/99194/files